src/virtualenv.py

```diff
@@ -1221,7 +1221,7 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
                 continue
             for fn in os.listdir(stdlib_dir):
                 bn = os.path.splitext(fn)[0]
-                if fn != "site-packages" and bn in REQUIRED_FILES:
+                if fn not in ("site-packages", "lib-dynload") and bn in REQUIRED_FILES:
```

Comment:
we should have some explanation attached here as a comment as why not link these 👍

Reply:
Yes, I'm still trying to figure out why the tests fail. I'll add a comment when I'm sure this is the solution.

Reply:
And no, that's not the right way to fix. I think I need to read the code thoroughly....

Reply:
I've merged in realpath, so try based on top of that?